### DISCUSSION TITLE
Discussion about blog post: firstpost

### DISCUSSION THREAD

**Discussion by ScriptRaccoon** [ScriptRaccoon/portfolio-website#1] [Announcements]
Let's discuss about https://scriptraccoon.dev/blog/firstpost

**Comment by ScriptRaccoon** [comment]
Nice, this works! I have used [giscus/svelte](https://github.com/giscus/giscus-component) to add this comment section below each post.
For this one, the comments are stored here: https://github.com/ScriptRaccoon/portfolio-website/discussions/1

**Comment by ScriptRaccoon** [comment]
Reactions to the post can be made as well. They are stored as reactions to the initial message in the discussion on GitHub. Nice solution.

**Comment by mxkaske** [comment]
🔥🔥

**Reply by ScriptRaccoon** [reply]
This is a reply
